cypress/integration/instance/InstanceEdit.ts

```diff
@@ -50,5 +50,65 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
             });
         });
     });
+
+    it("Switch from 'Webhook' to 'Ignore' Error handler", () => {
+      //Open 'Instance one' page
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId("Instance one", "PF4/TableRow")
+        .within(() => {
+          cy.get(
+            "td:first a[data-testid='tableInstances-linkInstance']"
+          ).click();
+        });
+
+      // Switch to error handling tab
+      cy.ouiaId("error-handling", "PF4/TabButton").click();
+
+      // Wait tab is loaded
+      cy.get(".pf-c-description-list__term").should("have.length", 5);
+      // Once tab is loaded start edit mode
+      cy.ouiaId("error-handling", "PF4/TabContent").within(() => {
+        cy.ouiaId("edit", "PF4/Button")
+          .should("have.attr", "aria-disabled")
+          .and("equal", "false");
+        cy.ouiaId("edit", "PF4/Button").click();
+      });
+
+      // Change error handling strategy to 'Ignore'
+      cy.ouiaId("error-handling-method-selector", "PF4/Select").within(() => {
+        cy.get("button").should("not.have.attr", "disabled");
+      });
+      cy.ouiaId("error-handling-method-selector", "PF4/Select").click();
+      cy.get("button.pf-c-select__menu-item", { timeout: 10000 })
```

Comment:
In case that you want avoid this selector and use ouiaId. You can define it here:
https://github.com/jomarko/sandbox-ui/blob/MGDOBR-1095-test/src/app/Instance/ErrorHandling/ErrorHandlingSelection.tsx#L60-L63
https://github.com/jomarko/sandbox-ui/blob/MGDOBR-1095-test/src/app/Instance/ErrorHandling/ErrorHandlingSelection.tsx#L72-L74

These function components are not OUIA compilant so you have to use whole names of OUIA attributes (`data-ouia-component-id` and `data-ouia-component-type`).

Reply:
thank you for a tip, will do in a separate commit

Reply:
see 4b871ec